spec/std/option_parser_spec.cr

```diff
@@ -85,6 +85,105 @@ describe "OptionParser" do
     expect_capture_option ["-f", "-g -h"], "-f FLAG", "-g -h"
   end
 
+  describe "Consumption of flags following an ungiven optional argument" do
+    context "Given a short option with an optional value" do
+      it "doesn't eat a following short option" do
+        flag = nil
+        not_eaten = [] of String
+        args = ["-f", "-g", "-i"]
```

Comment:
You could use `%w` sygil for brevity:

```suggestion
        args = %w[-f -g -i]
```

ditto elsewhere

Reply:
I'm just trying to keep as close to the style in the rest of the file as possible.